pandas/tests/io/test_sql.py

```diff
@@ -64,6 +64,11 @@
 ]
 
 
+def table_uuid_gen(prefix: str) -> str:
+    """Generate a unique table name with context prefix."""
```

Comment:
You can remove this docstring - it doesn't add much value and we don't use docstrings in the test suite